### PR TITLE
Igal/podmam

### PR DESCRIPTION
**Pull request by tsorya**

### CODE REVIEW

File: skipper/runner.py [hunk 1]
```diff
@@ -89,14 +89,27 @@ def _run_nested(fqdn_image, environment, command, interactive, name, net, volume
...
+            '%s:/opt/skipper/skipper-entrypoint.sh:rw' % utils.get_extra_file("skipper-entrypoint.sh"),
+        ])
+        if os.path.exists('/var/run/docker.sock'):
+            volumes.append('/var/run/docker.sock:/var/run/docker.sock:rw')
+        if os.path.exists('/var/lib/osmosis'):
+            volumes.append('/var/lib/osmosis:/var/lib/osmosis:rw')
+    else:
+        volumes.extend([
+            '%(workspace)s:%(workspace)s:rw,Z' % dict(workspace=workspace),
+            '%(homedir)s/.netrc:%(homedir)s/.netrc:ro' % dict(homedir=homedir),
```
Comment (eranco74) [inline skipper/runner.py:106]: Some of these seems the same for all cases.
start a volumes array with them and than extend it in the if/else

File: skipper/runner.py [hunk 2]
```diff
@@ -124,9 +143,23 @@ def handle_volumes_bind_mount(docker_cmd, homedir, volumes, workspace):
     return docker_cmd
 
 
+# We have couple of special case mounts
+# 1. gitconfig file - it is required by skipper but may not exists, we don't want to create folder if it doesn't exist
+# that's why we create it as file
+# 2. .docker/config.json - if it is required and doesn't exists we want to create is as file with {} as data
+def _special_case_mounts(volume):
```
Comment (eranco74) [inline skipper/runner.py:150]: Consider renaming (create_volume_local_path_if_not_exists)

File: skipper/runner.py (hunk 1 above)
Comment (tsorya) [reply skipper/runner.py:106]: yap right

File: skipper/runner.py (hunk 2 above)
Comment (tsorya) [reply skipper/runner.py:150]: but it is relevant only for 2 cases

Comment (tsorya) [reply skipper/runner.py:150]: i will move some logic there and rename

**Pull request merged**